conf/janus.jcfg.sample.in

```diff
@@ -300,6 +300,7 @@ nat: {
 	#turn_rest_api = "http://yourbackend.com/path/to/api"
 	#turn_rest_api_key = "anyapikeyyoumayhaveset"
 	#turn_rest_api_method = "GET"
+	#turn_rest_api_timeout = 10
```

Comment:
The new property is not discussed in the text above. It should be explained that there's a configurable timeout, and that it's in seconds. As it is, people reading the file will just see `10` and not know what it means.